src/app/zap-templates/zcl/data-model/silabs/ami.xml

```diff
@@ -201,8 +201,8 @@ limitations under the License.
     <item value="0xA2" name="BatteryCoverRemoved"/>
     <item value="0xA3" name="BatteryCoverClosed"/>
     <item value="0xA4" name="ExcessFlow"/>
-    <item value="0xC0" name="CreditOk"/>
-    <item value="0xC1" name="LowCredit"/>
+    <item value="0xB0" name="CreditOk"/>
+    <item value="0xB1" name="LowCredit"/>
```

Comment:
Again, these should not be getting changed: they are defined in a spec somewhere.  If they are causing problems and are not part of Matter, they should be removed, as needed.

Reply:
https://github.com/project-chip/connectedhomeip/pull/20659